app/models/chargeback.rb

```diff
@@ -1,4 +1,6 @@
 class Chargeback < ActsAsArModel
+  singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)
```

Comment:
> when I redefine method like it here in chargeback.rb, it works 
> https://github.com/ManageIQ/manageiq/blob/master/lib/extensions/descendant_loader.rb#L251

What do you mean?

I don't like `singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader`.
It feels bad.  We can do it but it's not clear at all what it's doing or why it's required.